bosh-stemcell/spec/stemcells/cis_spec.rb

```diff
@@ -49,9 +48,27 @@
   end
 
 
-  context "For all infrastructure except Azure and Cloudstack", {exclude_on_azure:true, exclude_on_cloudstack:true} do
+  context "For all infrastructure except Azure and Cloudstack and FIPS", {exclude_on_azure:true, exclude_on_cloudstack:true, exclude_on_fips: true} do
     it 'confirms that all CIS test cases ran' do
-      expect($cis_test_cases.to_a).to match_array(base_cis_test_cases + ['CIS-2.24'])
+      expect($cis_test_cases.to_a).to match_array(base_cis_test_cases + ['CIS-2.24', 'CIS-8.1.12' ])
+    end
+  end
+
+  # not sure if include is a thing
+  context "For FIPS stemcells", {
+    exclude_on_alicloud: true,
+    exclude_on_aws: true,
+    exclude_on_azure: true,
+    exclude_on_google: true,
+    exclude_on_vcloud: true,
+    exclude_on_vsphere: true,
+    exclude_on_warden: true,
+    exclude_on_openstack: true,
+    exclude_on_cloudstack: true,
+    exclude_on_softlayer: true,
+  } do
+    it 'confirms that all CIS test cases ran' do
+      expect($cis_test_cases.to_a).to match_array(base_cis_test_cases + ['CIS-2.24', ])
```

Comment:
why we have comma after `'CIS-2.24'` but no value? Did we forget to add a CIS case here?